### PR TITLE
Implement registrant form wireframe pages

### LINKED ISSUE #231: Implement frontend views from wireframes in the Django application
_Outcome_: Content and proposed views from #190 for the new registrant form are implemented in the Django application. 

This does not require functionality to manipulate Application state which requires #221 and #222 

Views to implement:
- [x] Type of organization  
- [x] Organization name and mailing address
- [x] Authorizing official
- [x] Website for your organization
- [x] .gov domain
- [x] Purpose of your domain
- [x] Contacts for your domain
- [x] Other contacts for your domain
- [x] Security email for public use
- [x] Anything else we should know?
- [x] Requirements for registration and operation of .gov domains
- [x] Review and submit your domain request

- [ ] Testing

### PR DESCRIPTION
**Pull request by igorkorenfeld**
## 🗣 Description ##

This PR implements the views from the one question per page version of the wireframes [(v0.2B)](https://www.figma.com/proto/ninNjo0bTg7XfB4mvTp4hG/GetGov%2FWireframes?page-id=1057%3A4846&node-id=1352%3A11320&viewport=-652%2C727%2C0.43&scaling=min-zoom&starting-point-node-id=1352%3A11320).

Closes #231 

## 💭 Motivation and context ##
- Following the one question per page version, pages that previously had conditionally revealed questions have now been split into multiple pages
- I have put the content that will be dependent on the type of entity into separate includes files, as one idea of how this might be implemented. I imagine the logic for which content should be rendered will exist somewhere, and will pull in the appropriate partial html file as an include (this primarily relates to the examples on `Administrative Official` view and on the `.gov domain` view).

### Limitations and things not done
- No field validation, beyond checking that a required field has something in it. So inputs like telephone fields and zip codes currently accept all characters
- All buttons beyond the "Next" button are non-functional
- No conditional logic for questions or views
- Sidebar locks all steps ahead of the current step, even if the user has already completed them and just went back
- Review page data values are just placeholders

### CODE REVIEW

**Review by neilmb — APPROVED** [review]
This is spectacular to see all together. Lots of very minor suggestions, overall very nice.

For follow-on work, we should also add additional testing to `FormTest` for each of these pages. As we add field verification, we should also add passing and failing tests for them.

Great work @igorkorenfeld!

**Comment by igorkorenfeld** [comment]
Thanks for all the great feedback @neilmb !

File: src/registrar/templates/base.html [hunk 1]
```diff
@@ -171,10 +171,8 @@
 
     {% block section_nav %}{% endblock %}
 
-    <main id="main-content" class="grid-container">
```
[the remark this answers is not in the document]
Comment (igorkorenfeld) [reply src/registrar/templates/base.html:174]: That's a great point and thank you for catching that. My motivation for moving it out of here was that on any pages that contain a sidenav like the form, I think we want "Skip to main content" to skip to the form content and not the sidenav.

I've gone through and added a `main` element to all the existing `.html` templates where relevant. One alternative could be having two different base templates, one where there is a sidebar and one without, and each of those bases would have `main` element. I am not sure if that would be a better approach?

File: src/registrar/forms/application_wizard.py [hunk 2]
```diff
@@ -57,34 +64,239 @@ class OrganizationForm(forms.Form):
...
 
 
 # List of forms in our wizard. Each entry is a tuple of a name and a form
 # subclass
 FORMS = [
     ("organization", OrganizationForm),
-    ("contact", ContactForm),
+    ("organization_federal", OrganizationFederalForm),
+    ("organization_election", OrganizationElectionForm),
+    ("org_contact", OrgContactForm),
```
[the remark this answers is not in the document]
Comment (igorkorenfeld) [reply src/registrar/forms/application_wizard.py]: That's a good point and as a heads up I've also renamed `organization` to `organization_type` to make things clearer

**Comment by igorkorenfeld** [comment]
@neilmb I've added in the test for each of the pages so this is now ready for re-review. Thank you!

**Review by neilmb — APPROVED** [review]
That test case is amazing. Thanks for this @igorkorenfeld

**Pull request merged**